### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by mrguen**
Hi,
I am sorry all the lines seems changed because of line ending differences. Tell me how you generate your files so I might try to avoid this problem. I work on Windows 7 / textPad and upload on the web interface.

Sincerely

### CODE REVIEW

File: LowPower.cpp [hunk 1]
```diff
@@ -1,28 +1,30 @@
 /*******************************************************************************
 * LowPower Library
-* Version: 1.70
-* Date: 04-06-2018
+* Version: 1.80
+* Date: 04-oct-2018
```
Comment (rocketscream) [inline LowPower.cpp]: Please change it to Date: 04-10-2018 to streamline with previous format.

File: LowPower.cpp [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
 *
 * This is a lightweight low power library for Arduino.
 *
-* This library is licensed under Creative Commons Attribution-ShareAlike 3.0 
+* This library is licensed under Creative Commons Attribution-ShareAlike 3.0
 * Unported License.
 *
 * Revision  Description
 * ========  ===========
+* 1.80      Added support for ATmega88 and ATmega168P. powerExtStandby() modified because not supported on Atmega88 / Atmega168
```
Comment (rocketscream) [inline LowPower.cpp]: Please use space instead of tabs. Limit any line of code to 80 columns.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,14 @@
 ### Low-Power
 Lightweight low power library for Arduino.
 
-Version: 1.70
+Version: 1.80
 
-Date: 04-06-2018
+Date: 04-oct-2018
```
Comment (rocketscream) [inline README.md]: Again stick to the previous format of DD-MM-YYYY.

**Comment by mrguen** [comment]
I changed the date format. I haven't seen tabs so I hope this is okay.

**Comment by rocketscream** [comment]
Merged as v1.80 and thank you for contributing!

**Comment by mrguen** [comment]
My pleasure. I need it for my PowMeter project<https://pandauino.com/en/powmeter-shield-for-nano/>. Thanks for this handy library.


________________________________
De : rocketscream <notifications@github.com>
Envoyé : 4 octobre 2018 09:41
À : rocketscream/Low-Power
Cc : Thierry GUENNOU; Author
Objet : Re: [rocketscream/Low-Power] Add files via upload (#72)


Merged as v1.80 and thank you for contributing!

**Pull request merged**